pom.xml

```diff
@@ -360,6 +367,23 @@
           <autoReleaseAfterClose>true</autoReleaseAfterClose>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.jsonschema2pojo</groupId>
+        <artifactId>jsonschema2pojo-maven-plugin</artifactId>
+        <version>1.1.1</version>
+        <configuration>
```

Comment:
Can you add also the following configurations?

https://joelittlejohn.github.io/jsonschema2pojo/site/1.1.1/generate-mojo.html#includeJsr303Annotations
https://joelittlejohn.github.io/jsonschema2pojo/site/1.1.1/generate-mojo.html#useCommonsLang3
https://joelittlejohn.github.io/jsonschema2pojo/site/1.1.1/generate-mojo.html#useOptionalForGetters
https://joelittlejohn.github.io/jsonschema2pojo/site/1.1.1/generate-mojo.html#usePrimitives

Reply:
These four settings are `false` by default, but I would prefer to have them enabled.

Reply:
Actually, I am sorry, but I did not know that `useCommonsLang3` is a deprecated option (see for more information: https://github.com/joelittlejohn/jsonschema2pojo/issues/996). Can you delete this single line from `pom.xml`? This is the last change and I will merge the PR.